src/common/transformations/src/transformations/op_conversions/convert_gatherND8.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_gatherND8.hpp"
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include "itt.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(pass::ConvertGatherND8ToGatherND5, "ConvertGatherND8ToGatherND5", 0);
+
+pass::ConvertGatherND8ToGatherND5::ConvertGatherND8ToGatherND5() {
+    MATCHER_SCOPE(ConvertGatherND8ToGatherND5);
+
+    auto gather_nd_v8_pattern = ngraph::pattern::wrap_type<opset8::GatherND>();
+
+    matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        auto gather_nd_v8_node = std::dynamic_pointer_cast<opset8::GatherND>(m.get_match_root());
```

Comment:
Let's also check if batch_dims is equal to 0

Reply:
Done. Thank you!